src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js

```diff
@@ -81,14 +63,14 @@ const FACILITY_SIDEBAR_QUERY = `
 
 let compiledQuery = '';
 
-FACILITY_MENU_NAMES.forEach(facilityMenuName => {
+getsideNavsViaGraphQL.forEach(facilityMenuName => {
```

Comment:
There are couple of issues here:

1. We need to _invoke_ `getsideNavsViaGraphQL`
1. We need to pass buildOptions into it, but those are only available in src/site/stages/build/index.js.

I'm pushing a commit to address these issues.